release-notes/unreleased/2875-Localization-Issues-at-poi-filter.yml

```diff
@@ -0,0 +1,6 @@
+issue_key: 2875
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - ios
+  - android
+en: Fixed issue with TextAlignment for RTL languages at the opening hours SubTitle
```

Comment:
🙃 Can you rephrase that for our users instead of our developers?
But also, I think this should be shown in the stores because it is again something users can notice.